bin/bazel_status

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```

Comment:
This file is the bridge between Bazel and the current setup for stamping the binary and the image org/tag. It is used by Bazel with `--workspace_status_command=bin/bazel_status`. Ref https://docs.bazel.build/versions/master/user-manual.html#flag--workspace_status_command.